### ISSUE TITLE
Tab order should be updated when tabs change position on the GUI

### ISSUE THREAD

**Issue by sesteves** [jimradford/superputty#580]
It is possible to change the order of tabs by just dragging them. However the order that is followed when switching from a tab to another (using shortcut keys for example) is the creation order and not the order by which tabs are displayed on the GUI.
